src/main/java/core/mate/academy/producers/TruckProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.producers;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer {
+    @Override
+    public List get() {
+        Truck truck1 = new Truck("volvo", "pink", "yellow", 1567);
```

Comment:
As I understood from condition of exercise, MachineProducer implementations method get() should return the list of specific machines. For example: ` List<Truck> get()` . I mean, you return simply List, but it should be parameterized.